app/helpers/sessions_helper.rb

```diff
@@ -32,9 +32,26 @@ def logged_in?
     current_user.present?
   end
 
+  def current_user? user
+    user && user == current_user
+  end
+
+  def show_delete user
```

Comment:
```suggestion
  def show_delete? user
```
Mấy hàm trả về true/false nên thêm dấu ? phía sau nhé em